src/Morpho.sol

```diff
@@ -194,8 +194,8 @@ contract Morpho is IMorpho {
         Id id = marketParams.id();
         require(market[id].lastUpdate != 0, ErrorsLib.MARKET_NOT_CREATED);
         require(UtilsLib.exactlyOneZero(assets, shares), ErrorsLib.INCONSISTENT_INPUT);
-        // No need to verify that onBehalf != address(0) thanks to the authorization check.
         require(receiver != address(0), ErrorsLib.ZERO_ADDRESS);
+        // No need to verify that onBehalf != address(0) thanks to the authorization check.
```

Comment:
there is a good reason why it's here: it's the position of the onBehalf != address(0) check in other functions (where it is checked). And I'm not sure that putting it here improves anything.

Reply:
I do think it's easier to understand as a reader like so instead of expecting the reader to check the exact position of the corresponding check in other functions (and because the order of these checks doesn't impact the behavior here)

Reply:
I agree with @Rubilmax I think it's better to apply the change